src/API/API.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const rocketsAPI = axios.create({
+  baseURL: 'https://api.spacexdata.com/v3/rockets',
+  headers: { 'Content-type': 'application/json' },
+});
+
+const missionsAPI = axios.create({
+  baseURL: 'https://api.spacexdata.com/v3/missions',
+  headers: { 'Content-type': 'application/json' },
+});
+
+export { rocketsAPI, missionsAPI };
```

Comment:
We already have an APIs.js file so, adding another API.js file is not necessary, so, kindly remove the API.js file and add your logic in the APIs.js file.